### ISSUE TITLE
Add aliases for long commands

### ISSUE THREAD

**Issue by BreD1810** [profclems/glab#108]
**Is your feature request related to a problem? Please describe.**
It would be nice if a user could create aliases for long commands that they use a lot.

**Describe the solution you'd like**
The [GitHub CLI](https://cli.github.com/manual/gh_alias_set) handles aliases very nicely, so it may be a good idea to base it off of their approach. The key features include:
- Ability to list aliases
- Ability to add an alias
- Ability to delete an alias

Aliases should allow for insertion of arguments into their longer form. One way to do this would be a shell-style syntax such as:
```
glab alias set createissue 'glab issue create --title "$1" --description "$2"'
```
When the alias is used:
```
glab createissue "My Issue" "Something is broken."
```
It should be executed as:
```
glab issue create --title "My Issue" --description "Something is broken."
```

*Note: It is important that the user should be able to escape the dollar character (or whatever is used to indicate insertion). It is also important that the escape character can be escaped!*

**Describe alternatives you've considered**
Predefined aliases are nice (such as `glab issue create`/`glab issue new` being the same), but giving the user flexibility to create their own is much better.

**Comment by profclems** [comment]
Great! I thought so too. 
Do you think you could open a quick PR for that??

**Comment by BreD1810** [comment]
I might be able to pick it up soon. But if anyone wants to make a start on it, feel free :)

**Comment by profclems** [comment]
Resolved by #113
